plugin/keyring_vault/keyring_vault-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+ADD_EXECUTABLE(keyring_vault-test
```

Comment:
Not really. There is no command in Vault that would allow to delete all keys in a backend.